README.md

```diff
@@ -23,13 +23,26 @@ defines the library decisions and might tell you more about the priorities of
 the library. So below you can find the key design principles behind `relude`:
 
 1. **Productivity.** You can be more productive with a "non-standard" standard
-   library.
-2. **Avoid all** [**partial functions**](https://www.reddit.com/r/haskell/comments/5n51u3/why_are_partial_functions_as_in_head_tail_bad/)
-   (e.g. `head :: [a] -> a`). The types of partial functions lie about their
-   behavior and usage of such functions can lead to unexpected bugs and runtime
-   exceptions in pure code.
+   library, and `reludes` helps you with writing safer and more
+   efficient code faster.
+
+2. **Total programming**. Usage of
+   [**partial functions**](https://www.reddit.com/r/haskell/comments/5n51u3/why_are_partial_functions_as_in_head_tail_bad/)
+   can lead to unexpected bugs and runtime exceptions in pure
+   code. The types of partial functions lie about their behaviour. And
+   even if it is not always possible to rely only on total functions,
+   `reludes` strives to encourage best-practices and reduce the
+   chances of introducing a bug.
+
+  | __Partial__                     | __Total__                                  |
+  |---------------------------------|--------------------------------------------|
+  | `head :: [a] -> a`              | `head :: NonEmpty a -> a`                  |
+  | `tail :: [a] -> [a]`            | `tail :: NonEmpty a -> [a]`                |
+  | `read :: Read a => String -> a` | `readMaybe :: Read a => String -> Maybe a` |
+  | `fromJust :: Maybe a -> a`      | `fromMaybe :: a -> Maybe a -> a`           |
+
```

Comment:
Great table :heart: 

Do you think it is a good idea to add the same table to the Haddocks of the `Relude.Unsafe` module?
With the disclaimer, that maybe you can avoid unsafe function blah blah?

Reply:
@vrom911 I think this is an excellent idea :+1: